### PR TITLE
refactor(es2015ify the remaining):

### PR DESCRIPTION
**Pull request by sarbbottam**

### CODE REVIEW

File: src/inject.js [hunk 1]
```diff
@@ -122,20 +122,20 @@ function inject() {
 function attachClickInjecter() {
     let pixelBar = document.getElementById("js-pjax-loader-bar")
     const config = { attributes: true, childList: true, characterData: true, attributeOldValue: true };
-    ([].slice.call(document.getElementsByClassName("tabnav-tab")).forEach((tab) => {
+    document.getElementsByClassName("tabnav-tab").forEach((tab) => {
```
Comment (sarbbottam) [inline src/inject.js]: `forEach` is available on `NodeList` in `es2015`.

File: src/inject.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ function inject() {
             }
             if (todayValue && !stop) {
                 ++currentStreak
-            }else {
```
Comment (Naramsim) [inline src/inject.js]: I'm trying to follow these guidelines: https://github.com/airbnb/javascript#blocks--cuddled-elses

**Comment by sarbbottam** [comment]
@Naramsim Have I misinterpreted something?

**Comment by Naramsim** [comment]
Nono, I made a comment but I was wrong :) I deleted it.

**Comment by sarbbottam** [comment]
No worries, please let me know if it needs any update.

---

I haven't worked on extension earlier, are there any sorts of test strategy?  Found [this one](http://stackoverflow.com/questions/2869827/how-to-test-chrome-extensions), not quite sure though.

**Comment by Naramsim** [comment]
@sarbbottam 
`document.getElementsByTagName("div").forEach(() => {console.log("a")})`
In FF and Chrome gives me an error, and I cant find where in es6 you can loop over nodes with forEach
Unless you use the spread operator: `[...document.getElementsByTagName("div")].forEach(() => {console.log("a")})`

**Comment by sarbbottam** [comment]
@Naramsim you are right! I expected it to behave similar to `document.querySelectorAll`.

**Pull request merged**